app/src/main/java/app/getfeeling/feeling/MainActivity.kt

```diff
@@ -36,6 +36,10 @@ class MainActivity : DaggerAppCompatActivity() {
         mainNavController = findNavController(R.id.nav_host_fragment)
 
         setupBottomNavigationView()
+
+        if (!signInViewModel.isSignedIn()) {
+            mainNavController.navigate(R.id.action_me_fragment_to_sign_in_fragment)
```

Comment:
Samples I found seem to omit "fragment" but still name it in the format \<from\>\_to\_\<to\> see [here](https://github.com/android/architecture-components-samples/blob/master/NavigationBasicSample/app/src/main/res/navigation/navigation.xml). Naming it "action\_sign\_in\_redirect" (or something along those lines) doesn't seem quite right since that would go against the naming convention of all the other actions